src/tests/ftest/util/general_utils.py

```diff
@@ -304,3 +310,796 @@ def verify_rebuild(pool, log, to_be_rebuilt, object_qty, record_qty, errors=0):
+            self.connect()
+            self._call_method(self.pool.pool_query, {})
+            self.info = self.pool.pool_info
+
+    def check_pool_info(self, pi_uuid=None, pi_ntargets=None, pi_nnodes=None,
+                        pi_ndisabled=None, pi_map_ver=None, pi_leader=None,
+                        pi_bits=None):
+        """Check the pool info attributes.
+
+        Args:
+            pi_uuid (str, optional): pool uuid. Defaults to None.
+            pi_ntargets (int, optional): number of targets. Defaults to None.
+            pi_nnodes (int, optional): number of nodes. Defaults to None.
+            pi_ndisabled (int, optional): number of disabled. Defaults to None.
+            pi_map_ver (int, optional): pool map version. Defaults to None.
+            pi_leader (int, optional): pool leader. Defaults to None.
+            pi_bits (int, optional): pool bits. Defaults to None.
+
+        Returns:
+            bool: True if at least one expected value is specified and all the
+                specified values match; False otherwise
+
+        """
+        self.get_info()
+        checks = [
+            (key,
+             c_uuid_to_str(getattr(self.info, key))
+             if key == "pi_uuid" else getattr(self.info, key),
+             val)
+            for key, val in locals().items()
+            if key != "self" and val is not None]
+        return self._check_info(checks)
+
+    def check_pool_space(self, ps_free_min=None, ps_free_max=None,
+                         ps_free_mean=None, ps_ntargets=None, ps_padding=None):
+        """Check the pool info space attributes.
+
+        Args:
+            ps_free_min (list, optional): minimum free space per device.
+                Defaults to None.
+            ps_free_max (list, optional): maximum free space per device.
+                Defaults to None.
+            ps_free_mean (list, optional): mean free space per device.
+                Defaults to None.
+            ps_ntargets (int, optional): number of targets. Defaults to None.
+            ps_padding (int, optional): space padding. Defaults to None.
+
+        Returns:
+            bool: True if at least one expected value is specified and all the
+                specified values match; False otherwise
+
+        """
+        self.get_info()
+        checks = []
+        for key in ("ps_free_min", "ps_free_max", "ps_free_mean"):
+            val = locals()[key]
+            if isinstance(val, list):
+                for index, item in val:
+                    checks.append((
+                        "{}[{}]".format(key, index),
+                        getattr(self.info.pi_space, key)[index],
+                        item))
+        for key in ("ps_ntargets", "ps_padding"):
+            val = locals()[key]
+            if val is not None:
+                checks.append(key, getattr(self.info.pi_space, key), val)
+        return self._check_info(checks)
+
+    def check_pool_daos_space(self, s_total=None, s_free=None):
```

Comment:
(pylint-unused-argument) Unused argument 's_free'
(pylint-unused-argument) Unused argument 's_total'

Reply:
If not used, can be removed.

Reply:
Fixed pylint error